comps/third_parties/wav2lip/src/Dockerfile

```
@@ -29,18 +29,18 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missin
 RUN mkdir -p /home/user/comps
 COPY comps /home/user/comps
 COPY comps/third_parties/wav2lip/src/entrypoint.sh /usr/local/bin/entrypoint.sh
-
+# hadolint ignore=DL3003
 # Install ffmpeg with x264 software codec
-RUN git clone --depth=1 https://github.com/FFmpeg/FFmpeg.git /home/user/comps/animation/src/FFmpeg
-WORKDIR /home/user/comps/animation/src/FFmpeg
-RUN ./configure --enable-gpl --enable-libx264 --enable-cross-compile && \
-    make -j$(nproc-1) && \
+RUN git clone --depth 1 --branch ${REPO_VER} https://github.com/FFmpeg/FFmpeg.git /home/user/comps/animation/src/FFmpeg && \
+    cd /home/user/comps/animation/src/FFmpeg && rm -rf .git && \
```

IMHO clearer as:
```suggestion
WORKDIR /home/user/comps/animation/src
RUN git clone --depth 1 --branch ${REPO_VER} https://github.com/FFmpeg/FFmpeg.git && \
    cd FFmpeg && rm -rf .git && \
```

Note: FFmpeg may also add commit ID to its output, so it may be better to remove `.git` only after `make` invocation.

And same goes also for the other Dockerfile.